### ISSUE TITLE
Documentation: ActivityIndicator, props not shown correctly

### ISSUE THREAD

**Issue by DanielMSchmidt** [facebook/react-native#13699]
### Description

I opened [the docs page for ActivityIndicator](http://facebook.github.io/react-native/releases/0.43/docs/activityindicator.html#activityindicator). If you have a look at the props, they are not shown correctly, but `size?: [object Object], [object Object]` is rendered. There should be a representation of

```js
PropTypes.oneOfType([
      PropTypes.oneOf([ 'small', 'large' ]),
      PropTypes.number,
])
```

### Reproduction Steps and Sample Code

1. Visit http://facebook.github.io/react-native/releases/0.43/docs/activityindicator.html#activityindicator
2. See the props definition

### Solution

Using PropTypes instead of flow seems to lead to this problem, I will try to convert it to use flow. Otherwise we could try to improve [website/server/extractDocs](https://github.com/facebook/react-native/blob/master/website/server/extractDocs.js) to handle PropTypes better.

### Additional Information

* React Native version: 0.43
* Platform: web
* Development Operating System: MacOS
* Dev tools: none

**Comment by DanielMSchmidt** [comment]
So it turned out that switching to flow is no option, as these components use mixins. Will try to improve the doc generation instead

**Comment by DanielMSchmidt** [comment]
This seems to relate to [reactjs/react-docs#178](https://github.com/reactjs/react-docgen/issues/178), is solved in  `2.14`, we are on `2.9`. Updating may help, will try it

**Comment by hramos** [comment]
Hi there! This issue is being closed because it has been inactive for a while.

But don't worry, it will live on with Canny! Check out its new home: https://react-native.canny.io/feature-requests/p/documentation-activityindicator-props-not-shown-correctly
